### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
- Set new $YNH_ARCH helper
- Update requirements to 4.3
- Update config file
- Upgrade to Mattermost 6.2.1

### CODE REVIEW

**Comment by ericgaspar** [comment]
!tastemetitus

File: manifest.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
         "en": "Open source collaboration platform built for developers",
         "fr": "Plateforme de collaboration open source conçue pour les développeurs"
     },
-    "version": "6.1.0~ynh1",
+    "version": "6.1.0~ynh2",
```
Comment (kemenaran) [inline manifest.json:9]: As far as I understand, this is required only to force an update on people having already the previous `6.1.0~ynh1` version installed.

But in this PR we have only internal technical changes, and nothing that requires an update from the package users.

Should we remove this bump?

**Review by kemenaran — APPROVED** [review]

File: manifest.json (hunk 1 above)
Comment (ericgaspar) [reply manifest.json:9]: or we could wait the next version and merge to master the all thing.

**Comment by kemenaran** [comment]
Mattermost 6.2.1 is merged, let's go!

**Pull request merged**